edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/ConfigUpdater.cs

```diff
@@ -37,7 +42,9 @@ public async Task Init(IConfigSource configProvider)
             Preconditions.CheckNotNull(configProvider, nameof(configProvider));
             try
             {
-                configProvider.SetConfigUpdatedCallback(this.HandleUpdateConfig);
+                configProvider.ConfigUpdated +=
+                    async (sender, config) => await this.HandleUpdateConfig(config);
+
```

Comment:
I see some subscriptions but never unsubscribe to ConfigUpdated - do these things live till the end of the program together? (I am asking in the context of GC that it cannot clean up stuff when they referenced through event handlers)

Reply:
Good question. `ConfigUpdater` and `PolicyUpdateHandler` never unsubscribe and should receive events throughout the lifetime of EH core. `ConfigUpdater` was already "linked" with TwinConfigSource by the latter referencing a callback from the former. So I don't think that my change adds more to it.